test/systemtests/network_test.go

```diff
@@ -81,6 +83,36 @@ func (s *systemtestSuite) testInfraNetworkAddDelete(c *C, encap string) {
 	}
 }
 
+/* Confirm network inspect includes a dns endpoint (assumes this is the only endpoint on the network currently) */
+func (s *systemtestSuite) testNetworkInspectDNS(tenant, network string) (bool, error) {
+
+	netInspect, err := s.cli.NetworkInspect(tenant, network)
+	// Network inspect must succeed
+	if err != nil {
+		return false, err
+	}
+
+	//	logrus.Infof("   %+v   \n", netInspect)
+
+	// Network inspect should show exactly 1 endpoint
+	if len(netInspect.Oper.Endpoints) != 1 {
+		logrus.Infof("testNetworkInspectDNS has %d endpoints (should have 1)!", len(netInspect.Oper.Endpoints))
+		return false, errors.New("testNetworkInspectDNS has incorrect # of endpoints (should have 1)!")
+	}
+
+	// First endpoint in network inspect should not have a null container ID
+	if netInspect.Oper.Endpoints[0].ContainerID == "" {
+		return false, errors.New("testNetworkInspectDNS endpoint has null containerID!")
+	}
+
+	// First endpoint in network inspect should incude dns in the container name
+	if !strings.Contains(netInspect.Oper.Endpoints[0].ContainerName, "dns") {
```

Comment:
The reason i am suggesting the change is - its good to verify that the right skydns container is attached to the network/tenant. If we are validating skydns. We might as well validate the right skydns container is attached. We should use the same mechanism to test other dns containers (if and whenever we decide to do that) so that tenant name is embedded in the dns name.